platform/darwin/src/MGLPointCluster.h

```diff
@@ -0,0 +1,15 @@
+#import <Foundation/Foundation.h>
+#import "MGLFoundation.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+MGL_EXPORT
+// TODO: Should this be a MGLPointClusterFeature? Subclass of MGLPointFeature?
+@protocol MGLPointCluster <NSObject>
+@property (nonatomic, readonly, getter=isCluster) BOOL cluster;     // Keep so we match cluster def? Always YES.
+@property (nonatomic, readonly) uint32_t clusterId;
```

Comment:
Spell out “identifier” in symbol names, rather than abbreviating as `ID` or `Id`.